### ISSUE TITLE
Video does not play when using https

### ISSUE THREAD

**Issue by cm-chandler** [iwalton3/jellyfin-media-player#2]
When trying to play a video, video will not begin and freeze JMP. This issue is only seen when connected to the Jellyfin server with https through nginx. This issue is not seen when the same nginx config is modified and used without SSL and connected to via http.

I am using the documented nginx config without any of the extra options: https://jellyfin.org/docs/general/networking/nginx.html

```
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - ffmpeg: Opening https://<redacted>
2021-04-04 21:36:57 [ INFO  ] JS: nowplaying event: init 
2021-04-04 21:36:57 [ INFO  ] JS: Requesting url without automatic networking: https://<redacted>/Sessions/Playing 
2021-04-04 21:36:57 [ INFO  ] JS: nowplaying event: playbackstart 
2021-04-04 21:36:57 [ INFO  ] JS: nowplaying event: playbackstart 
2021-04-04 21:36:57 [ ERROR ] PlayerComponent.cpp @ 594 - ffmpeg: tls: failed to open CA file '/home/shinchiro/build64/install/mingw/etc/ssl/cert.pem': No such file or directory 
2021-04-04 21:36:57 [ ERROR ] PlayerComponent.cpp @ 594 - stream: Failed to open https://<redacted>
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - cplayer: Opening failed or was aborted: https://<redacted>
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - cplayer: Running hook: ytdl_hook/on_load_fail 
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - ytdl_hook: full hook  
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - ytdl_hook: No youtube-dl found with path youtube-dl.exe in config directories  
2021-04-04 21:36:57 [ ERROR ] PlayerComponent.cpp @ 594 - ytdl_hook:   
2021-04-04 21:36:57 [ ERROR ] PlayerComponent.cpp @ 594 - ytdl_hook: youtube-dl failed: not found or not enough permissions  
2021-04-04 21:36:57 [ DEBUG ] PlayerComponent.cpp @ 588 - cplayer: finished playback, loading failed (reason 4) 
2021-04-04 21:36:57 [ INFO  ] PlayerComponent.cpp @ 590 - cplayer:  
2021-04-04 21:36:57 [ INFO  ] PlayerComponent.cpp @ 474 - "Entering state: error (loading failed)" 
...
2021-04-04 21:36:57 [ INFO  ] JS: media element error: loading failed 
2021-04-04 21:36:57 [ INFO  ] JS: Uncaught ReferenceError: onErrorInternal is not defined 
```

**Comment by iwalton3** [comment]
Are you using self-signed HTTPS? It works fine on Windows with HTTPS that is trusted.

Self-signed HTTPS is not supported at this time. I would encourage using Let's Encrypt ideally.

**Comment by cm-chandler** [comment]
I am using Let's Encrypt, obtained with certbot.

![cert2](https://user-images.githubusercontent.com/81978586/113597959-4b6ca380-95f1-11eb-840e-400effd02b98.png)

I have just renewing my certs using the nginx certbot directions posted at https://jellyfin.org/docs/general/networking/letsencrypt.html and still see the issue.

I have also installed JMP on another windows machine connected to the same server with the same result.

**Comment by iwalton3** [comment]
Nevermind I can confirm this is definitely happening. The web client has a nasty habit of connecting insecurely to local IP addresses if it can, so I didn't catch this.

Will make a fixed release soon. It only affects Windows.

**Comment by iwalton3** [comment]
Fixed in v1.2.0

**Comment by cm-chandler** [comment]
Thank you!
